client/tests/integration/smartcontracts/mint_rose/Cargo.toml

```diff
@@ -1,10 +1,9 @@
 [package]
 name = "mint_rose"
+
+edition = "2021"
 version = "2.0.0-pre-rc.13"
```

Comment:
This is left here because we can't share workspace metadata in integration test crates, right?

Reply:
As you can see below it's not part of the workspace. But I will try to make a virtual manifest for these `smartcontract` tests and share from there

Reply:
I applied this